python/ray/__init__.py

```diff
@@ -24,6 +26,23 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if (platform.system().lower() == "Linux".lower()
+    and "Microsoft".lower() in platform.release().lower()):
+    # WSL's /proc/meminfo has a bug that crashes Python; work around it
+    try:
+        import psutil._pslinux
+    except ImportError:
+        psutil = None
+    psutil_open_binary = None
+    try: psutil_open_binary = psutil and psutil._pslinux.open_binary
+    except AttributeError: pass
+    if psutil_open_binary and psutil_open_binary.__name__ == "open_binary":
```

Comment:
why is the `== "open_binary"` check needed? hacks like these should definitely be thoroughly commented

Reply:
It's to make the change idempotent and ensure that if it's been patched by other code, we don't patch it again. I'll add a comment.